### PR TITLE
deploy BraveBlockScreenFingerprinting to 100% of Beta desktop users

### LINKED ISSUE #491: Enable BraveBlockScreenFingerprinting for Beta Desktop users
Now that Screen Fingerprinting is at 100% of users on Desktop Nightly, we can roll it out to Beta users. We did previously have this protection at 100% for Beta users, but then temporarily rolled it back while we uplifted a fix for https://github.com/brave/brave-browser/issues/27021. So my suggestion is to deploy for 100% of Beta users now, and then if everything looks good we can roll it out gradually in Release.

### PR DESCRIPTION
**Pull request by arthuredelstein**
Resolves #491

### CODE REVIEW

**Comment by kjozwiak** [comment]
We'll uplift/enable `BraveBlockScreenFingerprinting` on `BETA` once https://github.com/brave/brave-browser/issues/27492#issuecomment-1362096467 has been addressed.

**Comment by arthuredelstein** [comment]
I updated the PR -- we should be good to go now that https://github.com/brave/brave-browser/issues/27492 was fixed.

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers. QA has verified the PR on `main` (staging) as per https://github.com/brave/brave-variations/pull/562#issuecomment-1480614920. @bsclifton also verified that Nightly is working as expected.

**Pull request merged**
